src/hydra/dataProvider.js

```diff
@@ -334,11 +345,23 @@ export default (
         });
 
       case UPDATE:
+        const getUpdateHttpVerb = (params) => {
+          if (
+            params.data &&
```

Comment:
Couldn't we use the optional chaining here (Babel should probably transpile it correctly)?

Reply:
Great tip. Done.